### DISCUSSION TITLE
Install linter & formater as standalone tools

### DISCUSSION THREAD

**Discussion by eirnym** [biomejs/biome#3265] [General]
Biome provides some useful tools such as linter and formatter. How can I install them standalone without downloading using JavaScript package manager

**Comment by nhedger** [comment]
Our docs actually covers this. It depends on your platform.  https://biomejs.dev/guides/manual-installation/

**Reply by eirnym** [reply]
There's no manual build instructions neither other popular package managers

**Reply by ematipico** [reply]
You're more than welcome to help us to implement "other popular package managers"
